index.html

```diff
@@ -16,28 +16,33 @@
     <title>Mobile-First</title>
   </head>
   <body>
-    <header>
+    <header class="body-section">
       <main class="parent">
-        <header class="navbor">
+        <header class="navbar">
```

Comment:
[ ] Kindly fix the navbar design to match the provided Figma design.
your design 👇  :
![hernav](https://user-images.githubusercontent.com/62156607/230508878-a0a9e075-6b17-4a09-be24-0a6ea3883d5f.PNG)

Figma design👇  : 
![designnav](https://user-images.githubusercontent.com/62156607/230508896-943884c3-0e10-44e6-9dba-fcebd68eb196.PNG)